.github/workflows/e2e_tests.yml

```
@@ -4,139 +4,42 @@ on:
-        env:
-          URL: ${{ env.URL }}
-
-      # FIXME: This step is not working well. All environments seem to be Preview.
-      - name: Upload test results
-        if: ${{ steps.check.outputs.should_run == 'true' && failure() }}
-        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+          test-project: liam-app
+          url: ${{ github.event.deployment_status.target_url }}
+          working-directory: frontend/internal-packages/e2e
+
+      - name: liam-erd-sample-e2e_tests:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 34: mapping values are not allowed in this context

(syntax-check)